parser/shared/src/main/scala/hmda/parser/fi/panel/PanelCsvParser.scala

```diff
@@ -0,0 +1,33 @@
+package hmda.parser.fi.panel
+
+import javax.swing.JFileChooser
+import javax.swing.filechooser.FileNameExtensionFilter
+
+import scala.io.Source
+import scala.util.Try
+
+object PanelCsvParser extends App {
+  val file = pickFile()
+  val lines = file.getLines().toList.tail
+
+  for (line <- lines) {
+    val i = InstitutionParser(line)
+    println(i)
+  }
+
+  def pickFile(): Source = {
+    val chooser = new JFileChooser()
```

Comment:
The file chooser was just a temporary thing, I already had the code for it from an earlier attempt at parsing.  I was having issues with it though, so I might switch this back to a simple command line input.